src/bson.c

```diff
@@ -957,7 +979,7 @@ void php_phongo_bson_typemap_to_state(zval *typemap, php_phongo_bson_typemap *ma
 				map->array_type = PHONGO_TYPEMAP_NATIVE_OBJECT;
 			} else {
 				map->array_type = PHONGO_TYPEMAP_CLASS;
-				array_ce = zend_fetch_class(classname, classname_len, ZEND_FETCH_CLASS_AUTO TSRMLS_CC);
+				zend_class_entry *array_ce = zend_fetch_class(classname, classname_len, ZEND_FETCH_CLASS_AUTO TSRMLS_CC);
```

Comment:
Variable declarations need to be done first.
This won't compile on VS.. And I thought this was part of our --enable-developer-flags (-Wdeclaration-after-statement).